herddb-core/src/main/java/herddb/sql/CalcitePlanner.java

```diff
@@ -341,6 +341,84 @@ public TranslatedQuery translate(String defaultTableSpace, String query, List<Ob
+            query = query.substring(Arrays.stream(items).collect(Collectors.joining(" ")).length()).trim();
+            String tableSpace = "herd";
+            String tableName;
+            if (query.contains(".")) {
+                String tokens[] = query.split("\\.");
+                tableSpace = tokens[0].trim();
+                tableName = tokens[1].trim();
+            } else {
+                tableName = query.trim();
+            }
+            try {
+                Table t = manager.getTableSpaceManager(tableSpace).getTableManager(tableName).getTable();
+                if (t != null) {
+                    StringBuilder sb = new StringBuilder("CREATE TABLE "+tableSpace+"."+tableName);
+                    StringJoiner joiner = new StringJoiner(",","(", ")");
+                    for(Column c : t.getColumns()) {
+                        joiner.add(c.name + " "+ColumnTypes.typeToString(c.type));
+                    }
+                    if(t.getPrimaryKey().length > 0) {
+                        joiner.add("Primary key("+ Arrays.stream(t.getPrimaryKey()).collect(Collectors.joining(",")) + ")");
+                    }
+                    sb.append(joiner.toString());
+
+                    ValuesOp values = new ValuesOp(manager.getNodeId(),
+                            new String[]{"tabledef"},
+                            new Column[]{
+                                    column("tabledef", ColumnTypes.STRING)},
+                            Arrays.asList(
+                                    Arrays.asList(
+                                            new ConstantExpression(sb.toString())
+                                    )
+                            )
+                    );
+                    ExecutionPlan executionPlan = ExecutionPlan.simple(
+                            new SQLPlannedOperationStatement(values),
+                            values
+                    );
+                    return new TranslatedQuery(executionPlan, new SQLStatementEvaluationContext(query, parameters));
+                }
+
+            }catch (NullPointerException ex) {
+                ValuesOp values = new ValuesOp(manager.getNodeId(),
+                        new String[]{"error"},
+                        new Column[]{
+                                column("error", ColumnTypes.STRING)},
+                        Arrays.asList(
+                                Arrays.asList(
+                                        new ConstantExpression("Table not found")
+                                )
+                        )
+                );
+                ExecutionPlan executionPlan = ExecutionPlan.simple(
+                        new SQLPlannedOperationStatement(values),
+                        values
+                );
+                return new TranslatedQuery(executionPlan, new SQLStatementEvaluationContext(query, parameters));
+            }
+        }
+        ValuesOp values = new ValuesOp(manager.getNodeId(),
```

Comment:
Please throw an error, the client must see an hard error

Reply:
Makes sense. Will do.